python_modules/dagster/dagster/_core/remote_representation/external.py

```diff
@@ -528,6 +528,11 @@ def _job_index(self) -> JobIndex:
                 )
             return self._index
 
+    @property
+    def ref_tags(self) -> Mapping[str, str]:
```

Comment:
feel like we can come up with a name tied to what these are conceptually versus how its implemented 

the crux of this utility is that these are tags that have been copied over to the pointer object so you can know certain properties without having to load the full object 


* summary tags
* promoted tags
* critical tags
* pointer tags
* navigation tags
* preview tags 
* digest tags 

maybe preview is my favorite since these are tags available `pre view`ing the full object

Reply:
I like preview tags. Happy to make that change.